src/mlpack/methods/preprocess/preprocess_imputer_main.cpp

```diff
@@ -211,11 +209,8 @@ void BINDING_FUNCTION(util::Params& params, util::Timers& timers)
       }
     }
     timers.Stop("imputation");
-
-    if (!outputFile.empty())
-    {
-      Log::Info << "Saving results to '" << outputFile << "'." << endl;
-      Save(outputFile, input, false);
-    }
+    
+    if (params.Has("output"))
+        params.Get<arma::mat>("output") = std::move(output);
```

Comment:
I don't think this is correct---shouldn't this be `input` not `output` in the `std::move()`?